packages/salesforcedx-vscode-replay-debugger/src/core/logContext.ts

```diff
@@ -88,6 +88,16 @@ export class LogContext {
     );
   }
 
+  public meetsLogLevelRequirements(): boolean {
+    return (
+      this.logLines &&
+      this.logLines.length > 0 &&
+      this.logLines[0].match(
+        /\d{2}.*APEX_CODE,FINEST;.*VISUALFORCE,FINEST;.*/
```

Comment:
Regex for the log levels in the debug log. It's always the first line and categories are in alphabetical order.

Reply:
If I were to create the logs manually through the dev console, the highest level we can set visualforce to is FINER. I imagine this would cause some confusion if I don't know about the Turn On Logging for Replay Debugger command in VSCode and try to set the log levels to finest in the dev console. Should we check for FINER as well for Visualforce or should we change the dev console and make FINEST available?

Reply:
I'll change the check for Visualforce to FINER because there isn't any event in the FINEST category. 

Do you want me to also change the `Turn On Apex Debug Log for Replay Debugger` command to use FINER for Visualforce? If it stays at FINEST, I can go to the debug logs setup page and change the level to FINER and see no difference in events that are in FINER and FINEST.

Reply:
Yeah, I think changing it to FINER in `Turn On Apex Debug Log for Replay Debugger` would reduce confusion later on